compiler/damlc/tests/daml-test-files/Iou12.daml

```diff
@@ -140,7 +140,7 @@ main = scenario do
 
   -- some things are broken because we have had to hack signatories
   -- scheduled fix in DAML-LF for next week
```

Comment:
Does anybody remember what this is about? And when was "next week"?

Reply:
I don’t remember. Wouldn’t be surprised if this was before my time. Either way this comment seems super out of date so :fire: